Sprint 12

KJIACCuKA opened the pull request.

KJIACCuKA committed:
sprint_12

KJIACCuKA committed:
sprint_12

DanilaG commented on Image Feed/ImagesList/ImagesListModel.swift:
```diff
@@ -0,0 +1,39 @@
+    let welcomeDescription: String?
+    let thumbImageURL: String
+    let largeImageURL: String
+    let fullImageURL: String
```
Давай для хранения URL использовать тип данных URL

DanilaG commented on Image Feed/ImagesList/ImagesListModel.swift:
```diff
@@ -0,0 +1,39 @@
+    let fullImageURL: String
+    let isLiked: Bool
+    
+    init(result photo: PhotoResult) {
```
Сейчас, модель бизнес логики зависит от структуры ответа, что ухудшает тестируемость кода. Давай это исправим. Обычно в таких случаях делают какой-то мепир, или в структуре ответа делают мепинг в модель бизнеслогики

DanilaG commented on Image Feed/ImagesList/ImagesListModel.swift:
```diff
@@ -0,0 +1,39 @@
+struct Photo {
+    let id: String
+    let size: CGSize
+    let createdAt: String
```
Давай хранить дату используя тип `Date`

KJIACCuKA committed:
bug fix

KJIACCuKA committed:
Из-за багов, связанных со сторибордом - все приложение написано кодом.

Из-за возникающих багов, те участки приложения, которые были связаны со сторибордом - переписаны и написаны кодом. Замечания ревьюера исправлены

KJIACCuKA committed:
bug fix

KJIACCuKA committed:
fix URL force unwrap

KJIACCuKA committed:
bug fix
